draft-ietf-quic-transport.md

```diff
@@ -3246,7 +3246,21 @@ Implementors are encouraged to register version numbers of QUIC that they are
 using for private experimentation on the GitHub wiki at
 \<https://github.com/quicwg/base-drafts/wiki/QUIC-Versions\>.
 
-
+## Version Aliases
+
+In order to avoid ossification of the version number defined by this draft,
+servers announce a list of version numbers that they interpret as an alias for
+the version number used in this draft. Alias versions MUST NOT be a reserved
+version. Servers SHOULD send at least one version alias, and SHOULD frequently
+change the value that they announce. Each version alias contains a lifetime,
+which indicates how long the server will accept this version alias. It also
+contains an initial salt, which is used instead of the initial salt as defined
+in section 5.2 of {{QUIC-TLS}}. The list of version aliases is sent in the
+server's Transport Parameters (see {{transport-parameter-definitions}}).
+
+Clients SHOULD remember the aliases and use it for subsequent connections to the
+same server in the future. This applies to both 0-RTT connection as well as
+connections that don't use 0-RTT.
 
```

Comment:
We should add a note that Version Aliases must not be advertised during version negotiation

```suggestion
Version aliases MUST NOT be advertised in version negotiation Packets to avoid conflicts with future versions and experiments.
```